server/controllers/admin/enterprises.js

```diff
@@ -25,8 +25,8 @@ exports.list = function list(req, res, next) {
         BUID(location_id) AS location_id, logo, currency_id,
         gain_account_id, loss_account_id, enable_price_lock, enable_prepayments,
         enable_password_validation, enable_balance_on_invoice_receipt, enable_barcodes,
-        enable_auto_email_report, enable_index_payment_system, base_index_growth_rate,
-        posting_payroll_cost_center_mode, enable_require_cost_center_for_posting,
+        enable_auto_email_report, enable_index_payment_system, enable_activate_pension_fund, base_index_growth_rate,
+        posting_payroll_cost_center_mode, enable_require_cost_center_for_posting, transaction_type_id,
```

Comment:
I think we need to come up for a better name for `transaction_type_id`.  It doesn't explain what it is used for or where it is used.  Maybe call it `pension_transaction_type_id` or something to give the developer some indication of what it is being used for.